ultravox/tools/ds_tool.py

```diff
@@ -42,16 +37,18 @@ def map_split(self, ds_split: datasets.Dataset, num_proc: int) -> datasets.Datas
         )
 
     def _map_sample(self, sample):
-        text = sample[self.column_name]
+        # using a Jinja template for some added flexibility
+        # The {{ var }} syntax is how Jinja denotes variables
+        text = jinja2.Template("{{" + self.column_name + "}}").render(**sample)
         text = text["text"] if isinstance(text, dict) else text
         sample[self.audio_column_name] = tts_client.tts(text, self.voice)
         return sample
 
 
 @dataclasses.dataclass
 class TextGenerationTask:
-    new_column_name: str = simple_parsing.field(default="explanation", alias="-c")
-    template: str = simple_parsing.field(default=DEFAULT_TEXTGEN_TEMPLATE, alias="-T")
+    new_column_name: str = simple_parsing.field(alias="-c")
+    template: str = simple_parsing.field(alias="-T")
```

Comment:
Moved the existing template out and I didn't see a point in keeping the default.